test/end-to-end/processor/_normalizer_junit.xsl

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet exclude-result-prefixes="#all" version="2.0"
+    xmlns:local="x-urn:xspec:test:end-to-end:processor:normalizer:local"
+    xmlns:normalizer="x-urn:xspec:test:end-to-end:processor:normalizer"
+    xmlns:util="x-urn:xspec:test:end-to-end:processor:util"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+    
+    <!-- output serialization parameters should be in line with XSPEC_HOME/src/reporter/junit-report.xsl -->
+    <xsl:output name="junit" method="xml" omit-xml-declaration="yes" indent="yes"/>
```

Comment:
I think this `xsl:output` should be just `<xsl:output name="junit"/>`.
`XSPEC_HOME/src/reporter/junit-report.xsl` has `xsl:output`, but it has `name="escaped"` which is not referenced in `junit-report.xsl`. The serialization parameter is thus effectively `<xsl:output/>`.